MapboxNavigation/DayStyle.swift

```diff
@@ -21,10 +21,10 @@ extension UIColor {
     class var defaultLaneArrowSecondaryHighlighted: UIColor { get { return #colorLiteral(red: 1, green: 1, blue: 1, alpha: 1) } }
     
     class var trafficUnknown: UIColor { get { return defaultRouteLayer } }
-    class var trafficLow: UIColor { get { return defaultRouteLayer } }
-    class var trafficModerate: UIColor { get { return #colorLiteral(red: 0.9529411765, green: 0.6509803922, blue: 0.3098039216, alpha: 1) } }
-    class var trafficHeavy: UIColor { get { return #colorLiteral(red: 0.9137254902, green: 0.2, blue: 0.2509803922, alpha: 1) } }
-    class var trafficSevere: UIColor { get { return #colorLiteral(red: 0.5411764706, green: 0.05882352941, blue: 0.2196078431, alpha: 1) } }
+    class var trafficLow: UIColor { get { return #colorLiteral(red: 0.3215686275, green: 0.8784313725, blue: 0.3215686275, alpha: 1) } }//{ get { return defaultRouteLayer } }
```

Comment:
Unlike when displaying traffic everywhere on the map, low or unknown traffic along the route line isn’t necessary to call out explicitly, so we stick to the main route line color in that case. `trafficAlternateLow` is intended to represent the green traffic color, but it seems to be unused.